client/rest-high-level/src/main/java/org/elasticsearch/client/security/CreateApiKeyRequest.java

```diff
@@ -55,7 +56,7 @@ public CreateApiKeyRequest(String name, List<Role> roles, @Nullable TimeValue ex
         }
         this.roles = Objects.requireNonNull(roles, "roles may not be null");
         this.expiration = expiration;
-        this.refreshPolicy = (refreshPolicy == null) ? RefreshPolicy.getDefault() : refreshPolicy;
+        this.refreshPolicy = (refreshPolicy == null) ? RefreshPolicy.IMMEDIATE : refreshPolicy;
```

Comment:
Does't it make more sense to just leave it as null? The whole problem here is that client is trying to replicate behaviour that the server already doesn't. If the caller doesn't specify a refresh policy, then why do we go out of our way to pass one in the rest request?